core/src/edu/cornell/gdiac/temporary/entity/Note.java

```diff
@@ -27,172 +20,187 @@
 /**
  * Model class for enemy shells.
  */
-public class Note extends GameObject {
-	public static final float descentSpeed = -3.5f;
+public class Note{
+	public static final float descentSpeed = -4.5f;
```

Comment:
note descent should not be defined at the note level. it should be defined by the level or music level. also, shouldn't be final

Reply:
Intended to be changed after this PR